e2e-tests/gef/cypress/e2e/facility/unissued-facilities/unissued-facilities-bank-review-date.spec.js

```diff
@@ -0,0 +1,159 @@
+            });
+          });
+        });
+    });
+
+    beforeEach(() => {
+      cy.saveSession();
+      cy.login(BANK1_MAKER1);
+    });
+
+    describe('Visiting from unissued facilities page', () => {
+      beforeEach(() => {
+        cy.visit(relative(`/gef/application-details/${dealId}/unissued-facilities`));
+        unissuedFacilityTable.updateIndividualFacilityButton(0).click();
+
+        aboutFacilityUnissued.issueDateDay().type(dateConstants.threeDaysDay);
+        aboutFacilityUnissued.issueDateMonth().type(dateConstants.threeDaysMonth);
+        aboutFacilityUnissued.issueDateYear().type(dateConstants.threeDaysYear);
+
+        aboutFacilityUnissued.shouldCoverStartOnSubmissionNo().click();
+        aboutFacilityUnissued.coverStartDateDay().type(dateConstants.threeDaysDay);
+        aboutFacilityUnissued.coverStartDateMonth().type(dateConstants.threeDaysMonth);
+        aboutFacilityUnissued.coverStartDateYear().type(dateConstants.threeDaysYear);
+
+        aboutFacilityUnissued.coverEndDateDay().type(dateConstants.threeMonthsOneDayDay);
+        aboutFacilityUnissued.coverEndDateMonth().type(dateConstants.threeMonthsOneDayMonth);
+        aboutFacilityUnissued.coverEndDateYear().type(dateConstants.threeMonthsOneDayYear);
+
+        aboutFacilityUnissued.isUsingFacilityEndDateNo().click();
+
+        aboutFacilityUnissued.continueButton().click();
+      });
+
+      it('should redirect user to the about unissued facility page when clicking back link', () => {
+        cy.url().should('eq', relative(`/gef/application-details/${dealId}/unissued-facilities/${facilityOneId}/bank-review-date`));
+
+        bankReviewDate.backLink().click();
+
+        cy.url().should('eq', relative(`/gef/application-details/${dealId}/unissued-facilities/${facilityOneId}/about`));
+      });
+
+      it('should display error messages', () => {
+        bankReviewDate.bankReviewDateDay().clear().type('abcd');
+        bankReviewDate.continueButton().click();
+
+        bankReviewDate.errorSummary();
+        bankReviewDate.bankReviewDateError();
+      });
+
+      it('should redirect user to the unissued facility page when clicking continue', () => {
+        bankReviewDate.bankReviewDateDay().clear().type(dateConstants.todayDay);
+        bankReviewDate.bankReviewDateMonth().clear().type(dateConstants.todayMonth);
+        bankReviewDate.bankReviewDateYear().clear().type(dateConstants.todayYear);
+
+        bankReviewDate.continueButton().click();
+
+        cy.url().should('eq', relative(`/gef/application-details/${dealId}/unissued-facilities`));
+      });
+
+      it('should redirect user to the unissued facility page when clicking save and return', () => {
+        bankReviewDate.bankReviewDateDay().clear().type(dateConstants.todayDay);
+        bankReviewDate.bankReviewDateMonth().clear().type(dateConstants.todayMonth);
+        bankReviewDate.bankReviewDateYear().clear().type(dateConstants.todayYear);
```

Comment:
these could go in a form-submission describe and have the 3 repeated lines in a beforeEach